servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcExecutionStrategies.java

```diff
@@ -63,7 +63,9 @@ public static GrpcExecutionStrategy defaultStrategy() {
      * When merged with another execution strategy the result is always this strategy.
      *
      * @return {@link GrpcExecutionStrategy} that disables all request-response path offloads.
+     * @deprecated Use a custom strategy with no offloads instead.
```

Comment:
Currently they would have to use a custom builder strategy as there is no constant for `GrpcExecutionStrategies.offloadNone()`, only `HttpExecutionStrategy.offloadNone()`. I will change the comment to suggest `customStrategyBuilder().noOffloads().build()`

Reply:
Should we add `GrpcExecutionStrategies.offloadNone()` instead to be consistent with `HttpExecutionStrategies`?

Reply:
I would like to try to live without it, even though it introduces asymmetry between the two Execution Strategies. The gRPC version of `offloadNone()` would be used a lot less than the HTTP version. I have been wondering if `GrpcExecutionStrategy` should not be refactored to be more like `ConnectAndHttpExecutionStrategy` where it explicitly includes an HTTP strategy as well as any additional gRPC behaviour (empty for now) rather than only extending `HttpExecutionStrategy`.